src/plugins/imagery/mixins/imageryData.js

```diff
@@ -30,7 +30,7 @@ export default {
         this.timeSystemChange = this.timeSystemChange.bind(this);
         this.setDataTimeContext = this.setDataTimeContext.bind(this);
         this.setDataTimeContext();
-        this.openmct.objectViews.on('clearData', this.clearData);
+        this.openmct.objectViews.on('clearData', this.dataCleared);
```

Comment:
Hmm so we clear out the data locally, but do we need to re-load the telemetry collection? Or tell it to clear it's data somehow?

Reply:
@jvigliotta Thoughts?

Reply:
yeah, this is a good question. As noted, TC's don't have a clear method, they're just holding the telemetry and letting you know when new stuff is added or removed. So, I think in this case, the best option, would be to just ignore TC. The view itself has cleared the data and that should be enough for this issue. The data is still in the TC but it won't be emitted again, so we should be safe.

Reply:
The only issue I see, would be if you're acting on remove, which you are, but it's just a filter and at that point, there's nothing in the imageHistory. So should be fine.